extensions/das/artifacts/metering-dashboard/jaggery-files/member-status.jag

```diff
@@ -17,9 +17,55 @@
 (function () {
     var log = new Log();
 
-    var TABLE_NAME = "tableName";
+    var HTTPS_TRANSPORT = "https";
+    var AUTHORIZATION_HEADER = "Authorization";
+    var ANALYTICS_SERVICE = "/services/AnalyticsWebService";
+    var AUTHENTICATION_SERVICE = "/services/AuthenticationAdmin";
+    var AUTH_TOKEN = "authToken";
+    var LOCALHOST = "localhost";
+    var HTTP_USER_NOT_AUTHENTICATED = 403;
 
-    var HTTP_INTERNAL_ERROR = 500;
+    var carbon = require('carbon');
+    var configs = require('/configs/designer.json');
+    var HTTPConstants = Packages.org.apache.axis2.transport.http.HTTPConstants;
+    var AnalyticsWebServiceStub = Packages.org.wso2.carbon.analytics.webservice.stub.AnalyticsWebServiceStub;
+    var AuthenticationAdminStub = Packages.org.wso2.carbon.authenticator.stub.AuthenticationAdminStub;
+
+    var analyticsWSUrl = carbon.server.address(HTTPS_TRANSPORT) + ANALYTICS_SERVICE;
+    var authenticationWSUrl = carbon.server.address(HTTPS_TRANSPORT) + AUTHENTICATION_SERVICE;
+    var analyticsStub;
+    try {
+        analyticsStub = new AnalyticsWebServiceStub(analyticsWSUrl);
+    } catch(e) {
+        if (log.isDebugEnabled()) {
+            log.debug("Unable to instantiate AnalyticsWebServiceStub instance. This could be a CEP server. Returning");
+        }
+        return;
+    }
+    var serviceClient = analyticsStub._getServiceClient();
+    var options = serviceClient.getOptions();
+    options.setManageSession(true);
+
+    var authParam = request.getHeader(AUTHORIZATION_HEADER);
```

Comment:
We do not need to explicitly set it. This is how standard DAS gadgets work. Once the end user log into the DAS portal, the client will receive a session cookie. This cookie includes a session token/JSESSIONID. On the next call from the client to the API will have this. As a result the API call will get authenticated.